Harness.java

```diff
@@ -78,17 +80,22 @@ private void measurePermutation(List<Integer> permutation) {
     List<Integer> sorted = solution.pizzaSort(new ArrayList<>(permutation), comparator);
 
     for (int i = 0; i < permutation.size(); i++) {
-      if (sorted.get(i) != permutation.size()-i-1) {
+      int actual = sorted.get(i);
+      int expected = arrayLength - i - 1;
+      if (actual != expected) {
+        int difference = actual - expected;
         timesWrong[i]++;
+        sumOfSquaredDifferences[i] += (difference * difference);
```

Comment:
I'd also be interested in how far off a number is from its actual position, `sumOfSquaredDifferences[actual] += (difference * difference)` which would favour a list like 1,3,2,4,5,6 over 1,3,4,5,6,2 (position 2 is one off in both , but number 2 is 1 off in first list and 4 off in second).

AlecSolution is a good example of this because if it doesn't put #2 in the #2 spot it puts it in the #(n/2) spot